executor/adapter.go

```diff
@@ -132,7 +132,7 @@ func (a *recordSet) NewChunk() *chunk.Chunk {
 }
 
 func (a *recordSet) SupportChunk() bool {
```

Comment:
We can remove SupportChunk too.

Reply:
some other implementors of RecordSet have not supported chunk by now, we can not remove this function safely.